todo_app/data/session_items.py

```diff
@@ -63,7 +64,22 @@ def save_item(item):
     """
     existing_items = get_items()
     updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
-
+    updated_items.sort(key=lambda val: _SORT_ORDER[val['status']])
     session['items'] = updated_items
 
     return item
+
+def remove_item(id):
+    items = get_items()
+    items = [item for item in items if item['id'] != id]
+    print(items)
+    session['items'] = items
+    return
+
+def get_next_id():
+    items = get_items()
+    max_id = 0
+    for item in items:
+        if item['id'] > max_id:
+            max_id = item['id']
+    return max_id + 1
```

Comment:
Again minor, but worth comparing how we could also do this with the in-built max function combined with a list comprehension (i.e. a more functional approach):
```suggestion
    if items.length == 0:
        return 1
    maximum_existing_id = max([item['id'] for item in items])
    return maximum_existing_id + 1
```

That's probably less easy to read for non-Python developers, but is slightly more concise and helps avoid mutation which can make it a bit easier to reason about